modules/mobile/app/serializers/mobile/v0/community_care_provider_serializer.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class CommunityCareProviderSerializer
+      include FastJsonapi::ObjectSerializer
+
+      set_id :provider_identifier
+
+      attribute :name do |object|
```

Comment:
This was taken from the CCP serializer. IMO this should be a model method rather than live in the serializer. That would prevent me from having to copy their business logic.